base/bucket.go

```diff
@@ -32,32 +32,68 @@ const (
 )
 
 const (
-	GoCouchbase CouchbaseDriver = iota
-	GoCB
-	GoCBGoCouchbaseHybrid // Defaults to GoCB, falls back to GoCouchbase for missing functionality
+	GoCouchbase            CouchbaseDriver = iota
+	GoCB                                   // Use GoCB driver with default Transcoder
+	GoCBCustomSGTranscoder                 // Use GoCB driver with a custom Transcoder
```

Comment:
Yep that's been filed in https://github.com/couchbaselabs/sync-gateway-accel/issues/136